src/ChainSafe.GamingSdk.Gelato/Gelato.cs

```diff
@@ -28,13 +27,30 @@ public class Gelato : IGelato, ILifecycleParticipant
         public Gelato(IHttpClient httpClient, IChainConfig chainConfig, GelatoConfig config, IRpcProvider provider, ISigner signer, IContractBuilder contractBuilder)
```

Comment:
You can remove IRpcProvider from the list of arguments as well.
The same is true for the other constructor overload.